layouts/page/superpowers.html

```diff
@@ -0,0 +1,404 @@
+{{ define "hero" }}
+    <header class="header-hero shadow-2xl relative overflow-hidden">
+        <div class="absolute hero-clouds w-full z-0"><img src="/images/hero-clouds.svg" alt=""></div>
+        <div class="header-hero-items two-thirds relative container mx-auto z-10">
+            <div class="header-hero-item">
+                <h1>Modern Infrastructure with <span class="text-blue-300 font-semibold">Superpowers</span>.</h1>
+                <p class="text-xl text-purple-100 leading-relaxed">
+                    Pulumi 2.0 includes new capabilities for Delivery, Provisioning,
+                    Architecture, Testing, and Policy-as-code &mdash; on any cloud, using your
+                    favorite languages.
+                </p>
+                <div class="header-hero-actions">
+                    <a class="btn btn-lg" href="{{ relref . "/docs/get-started" }}">
+                        Get Started
+                    </a>
+                    <a class="btn btn-lg btn-orange-translucent" href="#launch">
+                        Watch the Launch Event
```

Comment:
Until the event is live, it feels like this is a slight bait-and-switch on the CTA. Should it say "Register for the Launch Event" for now? It's a bit more honest and at least you know right away that it's something in the future, rather than right now.

Reply:
Maybe "Join the Launch Event"? Register sounds like a big commitment...

Reply:
Or Catch the Launch Event, or Join Us for Launch.

Reply:
"Join Us for Launch" - nice and short